cmake/MacroHelper.cmake

```diff
@@ -161,27 +161,20 @@ endmacro()
 # =============================================================================
 # Run nocmodl to convert NMODL to C
 # =============================================================================
-macro(nocmodl_mod_to_cpp modfile_basename)
+macro(nocmodl_mod_to_cpp modfile_basename source_dir working_dir)
```

Comment:
nitpick. shouldn't working_dir be target_dir. (I realize the custom_command has a WORKING_DIR option)

Reply:
Done